docs/proposals/scheduling/policy-effetive-strategy/statization-of-propagation-policy.md

```diff
@@ -0,0 +1,594 @@
+  the ResourceTemplate will also follow the modified Policy to adjust the distribution states.
+
+## Motivation
+
+In order to facilitate a deeper understanding of the motivation, first take you into the real business scenario,
+there are two roles in this scenario:
+
+* Normal user: they have no knowledge of Karmada and don't want to learn how to use Karmada, 
+  and is only responsible for deploying resources of their own application in a Namespace.
+* Cluster administrator: a person who is familiar with Karmada and is responsible for maintaining Karmada CRD resources such as ClusterPropagationPolicy.
+
+The cluster administrator does not know which users, which namespaces, and which resources will be available in the future, 
+so he creates a global default ClusterPropagationPolicy to manage all resource templates.
+
+As normal users come on board, many resources are already deployed in the cluster federation.
+
+At this point, the cluster administrator has the following two types of requirements:
+
+* He needs to change the global ClusterPropagationPolicy in the future, for example, to add a new distribution cluster for all the resources of all the users, 
+  but he is afraid that the update of the ClusterPropagationPolicy will lead to a large number of workloads restarting, 
+  and even the restarting will fail to affect the user's business, which is a high operational risk. 
+  He would like to avoid the risk as a cluster administrator as much as possible.
+* In the future, there may be individual users who need a customized distribution policy, for example, 
+  if the global ClusterPropagationPolicy is configured for dual-cluster distribution, but an individual user's resources need triple-cluster distribution, 
+  the administrator would like to have a way to individually configure a customized distribution policy for the individual user.
+
+
+```mermaid
+graph LR
+A(fa:fa-user-cog Cluster Administrator) --> B[Create Global Default \n ClusterPropagationPolicy]
+B --> C{Change\nRequirements}
+C --> D[Update Default \n ClusterPropagationPolicy]
+C --> E[Customize Another \n ClusterPropagationPolicy]
+D --> F{{Demand: don't affect existing resource immediately}}
+E --> G{{Demand: take over batch of resources from Default CPP}}
+```
+
+## Proposal
+
+The current Policy mechanism has some limitations, we need to optimize the behavior of Policy.
+
+### Guideline
+
+Any workload is running fine, don't roughly change them!
+
+### Base Principle
+
+* PropagationPolicy / ClusterPropagationPolicy are just a bunch of API-based system functions, shouldn't actively cause any existing workload distribution changes in the system.
+* Any destructive changes should be triggered from the ResourceTemplate.
+* After ResourceTemplate changed, there must be a matching Policy for subsequent propagation. If there is no matching policy, it should be pending.
```

Comment:
What does `changed` mean? `resourceVersion`?

Reply:
actually, any filed expect `.status` nor label/annotation by karmada self, means `change`.

just like any `.spec` or most `.metadata` means `change`.

Reply:
So your real question is that although the modification of any field can trigger the new Policy to take effect, the modification is not based on your actual business needs, right?